src/riak_kv_wm_crdt.erl

```diff
@@ -403,6 +404,22 @@ produce_json(RD, Ctx=#ctx{module=Mod, data=RObj, include_context=I}) ->
                      Type, Value, get_context(RespCtx,I), ModMap),
     {mochijson2:encode(Body), RD, Ctx}.
 
+generate_etag(RD, Ctx=#ctx{module=Mod, data=RObj, method=Method})
+  when Method == 'GET'; Method == 'HEAD' ->
+    {{RespCtx, _}, _} = riak_kv_crdt:value(RObj, Mod),
```

Comment:
We need to talk about this. As I commented elsewhere, this CRDT context is not suitable as an etag. Imagine a Set CRDT that has had two items added, its context may be `[{actor1, 2}]` and it has a value of `[alice, bob]`. Now imagine I remove `bob`. The context is _still_ `[{actor1, 2}]` but the value is now `[bob]`. You can't use the CRDT context (and opaque binary term) as an ETag since the underlying value can change without the context changing.